docker-compose.yml

```diff
@@ -0,0 +1,104 @@
+version: '3.8'
+
+services:
+  db:
+    image: 'mysql:8.0.19'
```

Comment:
8.0.19 is an older version:

https://hub.docker.com/layers/library/mysql/9.0.1/images/sha256-40e3265a989949477b1c5c7386b99e957545636c5313dac7a849a0fcbb05d9a1?context=explore

Reply:
patching for house keeping done updating